pkg/controller/backup_controller.go

```diff
@@ -458,6 +465,24 @@ func getObjectStore(cloudConfig api.CloudProviderConfig, manager plugin.Manager)
 	return objectStore, nil
 }
 
+// TODO(nrb): Consolidate with other implementations
```

Comment:
@nrb @carlisia let's say that whichever PR of our 3 merges first will include a function with a new name, like @nrb has here, for getting an object store from a location.  The other 2 of us will then rebase and use that function.  The last person to merge can drop the old function, and rename the new one if they want (or not).  Work for everyone?

Reply:
That makes sense to me.

Reply:
Works for me!